Certified-Anonymous-Identities-in-Hyperledger-Fabric.md

```diff
@@ -0,0 +1,17 @@
+# Lab Name
+Certified Anonymous Identities in Hyperledger Fabric
```

Comment:
Normally the lab name and markdown filename is the repository name in git... This would create a rather unwieldy repository name :-)

If you don't mind that's fine...  Otherwise maybe do something like `Fabric-Certified-Anonymous-Identities.md` resulting to git repo `hyperledger-labs/fabric-certified-anonymous-identities` to at least may it a bit more compact ?

Reply:
Thanks! we'll do that right now

Reply:
@trbs  , we change .md name file and you'd to set DCO pass manually because is the only problem.  
After the DCO pass we will have our repository?
Thanks